### PR TITLE
Task-10 Generic Routes Validation

### PR DESCRIPTION
**Pull request by krishna-successiveTech**
Completed all the tasks mentioned in Task-10

### CODE REVIEW

File: src/controllers/trainee/Controller.ts [hunk 1]
```diff
@@ -11,19 +12,50 @@ class TraineeController {
...
+            },
+        ];
+        res.status(200).send(successHandler('Successfully fetch trainees', data));
+        console.log('Successfully get Trainees');
     }
 
-    public create(req: Request, res: Response) {
-        res.send('create fake data');
+    public create(req: Request, res: Response, next: NextFunction) {
+        console.log('>>>>>>>>>>>>>', req.body);
```
Comment (Rahul-Successive) [inline src/controllers/trainee/Controller.ts]: please remove the console from here.

File: src/libs/routes/errorHandler.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 export default function errorHandler(err, req, res, next) {
     console.log('inside errorHandler');
-    const status = err.status || 500;
-    const message = err.message || 'error';
-    const error = err.error || 'Not Found';
+    const status = err[0].status || 500;
```
Comment (Rahul-Successive) [inline src/libs/routes/errorHandler.ts]: use destructuring wherever possible

**Review by Rahul-Successive — CHANGES_REQUESTED** [review]

File: src/controllers/trainee/Controller.ts (hunk 1 above)
Comment (krishna-successiveTech) [reply src/controllers/trainee/Controller.ts]: Done

File: src/libs/routes/errorHandler.ts (hunk 2 above)
Comment (krishna-successiveTech) [reply src/libs/routes/errorHandler.ts]: Done

**Pull request merged**